modules/python/Makefile.security

```diff
@@ -0,0 +1,15 @@
+BANDIT=$(call which, bandit)
+BANDIT_FLAGS=--level --level 
+
+PYTHON_SECURITY_TARGETS?=$(shell find . -name "*.py" -not -path "*/.venv/*" -not -path "*/$(BUILD_HARNESS_VENV_NAME)/*" -not -path "*/.tox/*" -not -path "*/.egg*" -not -path "*/frontend/*" -not -path "*/dist/*" -not -path "*/build/*" -not -path "*/tests/*")
```

Comment:
Is there a reason not to use PYTHON_LINT_TARGETS? Perhaps we could rename it to `PYTHON_QUALITY_TARGETS` or something more generic.

Reply:
Changed to `PYTHON_LINT_TARGETS` since we are performing security linting